dpgen2/op/run_lmp.py

```diff
@@ -240,7 +219,9 @@ def lmp_args():
                 default=False,
                 doc=doc_shuffle_models,
             ),
-            Argument("head", str, optional=True, default=None, doc=doc_head),
+            Argument(
+                "model_frozen_head", str, optional=True, default=None, doc=doc_head
```

Comment:
```suggestion
                "model_frozen_head", str, optional=True, default=None, doc=doc_head, alias=["head"],
```

Reply:
This parameter is internal and passed within dpgen2 rather than for users. The name here is just for better clarity.